bwc-test/src/test/java/SecurityBackwardsCompatibilityIT.java

```diff
@@ -95,19 +102,16 @@ private void assertPluginUpgrade(String uri) throws Exception {
         Map<String, Map<String, Object>> responseMap = (Map<String, Map<String, Object>>) getAsMap(uri).get("nodes");
         for (Map<String, Object> response : responseMap.values()) {
             List<Map<String, Object>> plugins = (List<Map<String, Object>>) response.get("plugins");
-            Set<Object> pluginNames = plugins.stream().map(map -> map.get("name")).collect(Collectors.toSet());
-            switch (CLUSTER_TYPE) {
-                case OLD:
-                    Assert.assertTrue(pluginNames.contains("opendistro_security"));
-                    break;
-                case MIXED:
-                    Assert.assertTrue(pluginNames.contains("opensearch-security"));
-                    break;
-                case UPGRADED:
-                    Assert.assertTrue(pluginNames.contains("opendistro_security"));
-                    break;
+            Set<String> pluginNames = plugins.stream().map(map -> (String) map.get("name")).collect(Collectors.toSet());
+
+            final Version minNodeVersion = this.minimumNodeVersion();
+
+            if (minNodeVersion.major <= 1) {
+                assertThat(pluginNames, hasItem("opensearch_security"));
```

Comment:
did you mean `opendistro_security` here?

Reply:
These tests don't work because of the build breaks, when they do I'll make sure it checks the correct value.  Leaving this open until then